avocado/plugins/yaml_to_mux.py

```diff
@@ -310,7 +310,7 @@ def run(self, args):
             try:
                 data.merge(create_from_yaml(multiplex_files, debug))
             except IOError as details:
-                logging.getLogger("avocado.app").error(details.strerror)
+                logging.getLogger("avocado.app").error(details)
```

Comment:
You're exposing internal information:
```
[Errno 2] No such file or directory: 'mux.yaml'
```
This `[Errno 2]` is the internal exception error code. No need for the user to receive that information.